tests/test_utils/test_comparison.py

```diff
@@ -0,0 +1,82 @@
+"""Test the comparison functions."""
+
+from unittest import TestCase
+
+from sklearn.ensemble import RandomForestClassifier
+
+from molpipeline import Pipeline
+from molpipeline.any2mol import SmilesToMol
+from molpipeline.error_handling import ErrorFilter, FilterReinserter
+from molpipeline.mol2any import (
+    MolToConcatenatedVector,
+    MolToMorganFP,
+    MolToRDKitPhysChem,
+)
+from molpipeline.post_prediction import PostPredictionWrapper
+from molpipeline.utils.comparison import check_pipelines_equivalent
+
+
+def get_test_pipeline() -> Pipeline:
```

Comment:
do we have such a function already in the library somewhere? Maybe put it into the test utils

Reply:
Moved it to `tests/utils/default_models.py`.  
We have done something similar for Chemprop (see `test_extras/test_chemprop/chemprop_test_utils/default_models.py`)